### ISSUE TITLE
[Suggestion] QR code for shops

### ISSUE THREAD

**Issue by olympus12** [MohGovIL/hamagen-react-native#193]
I'd suggest that there would be an option for easier understanding of users who check into places.
Shops will place a printed QR code on it's entrance.

Users: allow to scan QR codes that will be placed on the entrance of the shop.

Shops: Generate unique QR codes to be printed and placed on it's entrance.

I think it could help track and follow easier sick covid19 users.

**Comment by dviryamin** [comment]
Maybe we need to make a separation 

**Hamagen application:**
- Share customer details(encrypted with MoH public key) with business using QR Code(E2E encryption between customer to business)

**Hamagen for Business application:**
- Generate QR Code and business registration
- Manual insert of IDs (For customers without smartphone)
- Keep customers data(encrypted with MoH public key)
- Share specific time-window with MoH when needed
- Self destruction of old data(more than 14 days for example)

This approach will keep the data decentralized and keep customers details safe.

**Comment by or81322** [comment]
What about adding a Health Declaration feature that will able the costumers to sign and submit it to the business.

**Comment by dviryamin** [comment]
Good idea, I think that The above features will bring more people to install Hamagen App and make it more relevant because recently people uninstall it.
